exercises/Neon-phenology-from-space-R.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: exercise
+topic: NEON
+title: Phenology from Space
+language: R
+---
+
+The [National Ecological Observatory Network](http://www.neonscience.org) has invested in [high-resolution airborne imaging](http://www.neonscience.org/data-resources/get-data/airborne-data) of their field sites. 
+Elevation models generated from [LiDAR](http://neondataskills.org/self-paced-tutorial/1_About-LiDAR-Data-Light-Detection-and-Ranging_Activity1/) can be used to map the topography and vegetation structure at the sites.
+This data gets really powerful when you can compare ecological processes across 
+sites. 
+[Download]({{ site.baseurl }}/data/Neon-airborne.zip) the elevation models for the [Harvard Forest](http://harvardforest.fas.harvard.edu/) (`HARV`) and [San Joaquin Experimental Range](http://www.fs.fed.us/psw/ef/san_joaquin/) (`SJER`) 
+and the plot locations for each of these sites. 
+Often, plots within a site are used as representative samples of the larger site 
+and act as reference areas to obtain more detailed information and ensure 
+accuracy of satellite imagery (*i.e., ground truth*).
+
+These high-resolution images can then be integrated with satellite imagery that 
+is gathered more frequently. We will use data collected from [MODIS](http://modis.gsfc.nasa.gov/), 
+which orbits the globe every 16 days. 
+One common ecological process that can be observed from space is [phenology](https://en.wikipedia.org/wiki/Phenology) (*or seasonal patterns*) of plants.
+Multi-band satellite imagery can be processed to provide a vegetation index of greenness called [NDVI](https://en.wikipedia.org/wiki/Normalized_Difference_Vegetation_Index). 
+NDVI values range from `-1.0` to `1.0`, where negative values indicate clouds, 
+snow, and water; bare soil returns values from `0.1` to `0.2`; and green vegetation returns values greater than `0.3`.
+Because these image data are rather large, download the [`HARV_NDVI`]({{ site.baseurl }}/data/HARV_NDVI.zip) and [`SJER_NDVI`]({{ site.baseurl }}/data/SJER_NDVI.zip) and place them in a folder with the NEON airborne data.
```

Comment:
> How does "Because these image data are rather large" relate to the rest of the sentence?

Mostly this is a note of concession to you. I wanted to have just one `zip` for the exercise, but GitHub limits file upload to 100MB.  Easy enough to remove the language if you are happy with there being multiple files. Otherwise, we can seek a different solution, like storing the data outside of GitHub.